docs/pages/activationcard.js

```diff
@@ -23,15 +23,15 @@ export default function ActivationCardPage({
           <MainSection.Card
             cardSize="md"
             type="do"
-            title="When to Use"
+            title="When to use"
```

Comment:
Can these unrelated changes be pulled out to a separate PR to make it easier to review the relevant changes?

Reply:
I'm going to leave these for now since this PR is already approved, but will do for next time